### DISCUSSION TITLE
[Question][Hw5] About tainting in conditionals

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#613] [Q&A]
I have a question about tainting in conditionals. For example, in "if (x<y) ~", if x was originally safe and y was tainted, what happens to x if we go this branch?

**Comment by RiceBiscuits** [comment]
If executing `x<y` can change the `x` value, and `y` can affect `x`, then `x` can be tainted.

**Comment by RiceBiscuits** [comment]
> If executing `x<y` can change the `x` value, and `y` can affect `x`, then `x` can be tainted.

We will consider only directly affected case.

**Comment by ee12ha0220** [comment]
Thank you!
